scss/_blankslate.scss

```diff
@@ -63,13 +63,21 @@
     border-radius: 3px;
   }
 
-  > .mega-octicon { color: #aaa; }
+  > .mega-octicon {
+    color: #aaa;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.